internal/impl/devbox.go

```diff
@@ -677,8 +678,9 @@ func (d *Devbox) installNixProfile() (err error) {
 	}
 
 	cmd.Env = nix.DefaultEnv()
-	cmd.Stdout = d.writer
-	cmd.Stderr = d.writer
+	cmd.Stdout = ux.NewFilterWriter(d.writer, "devbox-development")
```

Comment:
can we make the filtering more specific to:
```
replacing old 'devbox-development'
installing 'devbox-development'
```
?

I think it could be done by having `NewFilterWriter` take in a `[]string` as the second parameter.

The name `devbox-development` is also sometimes used in the paths in the store `/nix/store/<hash>-devbox-development/path/to/derivation-or-binary` which the packages we're installing will have.  I think we want to preserve that output.